src/olympia/abuse/tests/test_models.py

```diff
@@ -3136,6 +3118,27 @@ def test_notify_reviewer_decision_rejection_addon_already_disabled(self):
             not in mail.outbox[0].body
         )
 
+    def test_notify_reviewer_decision_legal_forward(self):
```

Comment:
This test probably needs a doc string as well.. why should the status be disabled? why do we expect no CinderJob at first? why do we expect those specific parameters on the sub test call?